src/SpectatorView.Native/SpectatorView.Compositor/Compositor/IFrameProvider.h

```diff
@@ -11,10 +13,19 @@ class IFrameProvider
     {
         BlackMagic,
         Elgato,
+        AzureKinect_DepthCamera_Off,
+        AzureKinect_DepthCamera_NFOV,
```

Comment:
>AzureKinect_DepthCamera_NFOV [](start = 8, length = 28)

do we want to expose binned and unbinned?

Reply:
it kind of seems like we may want to expose rgb and depth as separate providers. Or I can imagine a world where you may want to use azure kinect + dslr. 

---
In reply to: [384184940](https://github.com/microsoft/MixedReality-SpectatorView/pull/327#discussion_r384184940) [](ancestors = 384184940)

Reply:
I didn't notice a quality difference in binned versus unbinned for NVOF, and unbinned WFOV is not supported at 30fps.

---
In reply to: [384185353](https://github.com/microsoft/MixedReality-SpectatorView/pull/327#discussion_r384185353) [](ancestors = 384185353,384184940)